src/plugins/telemetry/server/telemetry_collection/ingest_solutions/constants.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export const INGEST_SOLUTIONS_ID = 'ingest_solutions';
+
+// TODO: Ideally this list should be updated from an external public URL (similar to the newsfeed)
```

Comment:
A reasonable suggestion but I question if the external URL should be public or internal to Elastic. Perhaps it will suffice to add a public method to the telemetry plugin to update these?

Reply:
Any reason in particular why the URL should be internal to Elastic only? 

Personally, I don't see a problem with the list being public (read-only). This repo is public and has an initial shipped build. Making the URL internal to Elastic would complicate the accessibility from already deployed builds.

Am I missing any other possible concerns?

Reply:
A public API for a `read-only` list is fine, my concern was over  _write_ access.